stdpopsim/cli.py

```diff
@@ -96,7 +96,7 @@ def get_models_help(species_id, model_id):
         models_text += indent + "Populations:\n"
 
         for population in model.populations:
-            if population.allow_samples:
```

Comment:
Let's keep this as ``allow_samples``, it's more  readable and it means we don't need to know what the exceptional value is.

Reply:
Done